target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -40,6 +40,9 @@ ipq40xx_setup_interfaces()
 	compex,wpj428)
 		ucidef_set_interface_lan "eth0 eth1"
 		;;
+	engenius,ens620ext)
```

Comment:
Looks like this could fit with the 8dev,jalapeno and alfa-network,ap120c-ac devices. Please add your device there.

Reply:
Fixed in next commit.